Source/Fuse.Controls.CameraView/Android/CameraImpl.java

```diff
@@ -253,6 +257,164 @@ public void setFlashMode(String flashMode) {
+                        break;
+                    case 2: //portrait upsidedown
+                        rect = new Rect(top, left, top + focus_area_size, left + focus_area_size);
+                        break;
+                    case 1: //landscape left
+                        rect = new Rect(left, top, left + focus_area_size, top + focus_area_size);
+                        break;
+                    default:
+                        rect = new Rect(left, top, left + focus_area_size, top + focus_area_size);
+                        break;
+                }
+
+
+                List<Camera.Area> meteringAreas = new ArrayList<Camera.Area>();
+                meteringAreas.add(new Camera.Area(rect, 1000));
+                parameters.setFocusAreas(meteringAreas);
+
+                if (focusModes.contains(Camera.Parameters.FOCUS_MODE_AUTO)) {
+                    parameters.setFocusMode(Camera.Parameters.FOCUS_MODE_AUTO);
+                }
+
+                if (parameters.getMinExposureCompensation() != 0 
+                    && parameters.getMaxExposureCompensation() != 0
+                    && parameters.isAutoExposureLockSupported()
+                ) {
+                    
+                    int exposureAmount = (parameters.getMinExposureCompensation() - parameters.getMaxExposureCompensation()) * -1;
+                    int designAdjustment = 1; //added 1 to account for a control panel covered area on the bottom of the camera that can't be tapped on for focus
+                    if ((y/cameraHeight) < 0.5) { //apply only to upper half of camera screen
+                        designAdjustment = 0;
+                    }
+
+                    int amountApplied = (int)Math.round( (exposureAmount * (y/cameraHeight)) ) + designAdjustment; 
+                    amountApplied = (amountApplied > exposureAmount) ? exposureAmount : amountApplied;
+                    
+                    parameters.setExposureCompensation(parameters.getMinExposureCompensation() + amountApplied);
+                    
+                    parameters.setAutoExposureLock(false);
+                }
+
+
+                _camera.setParameters(parameters);
+
+
+
+                //check for lock
+                if (isFocusLocked == 1) {
+
+                    if (focusModes.contains(Camera.Parameters.FOCUS_MODE_MACRO)) {
+                        parameters.setFocusMode(Camera.Parameters.FOCUS_MODE_MACRO);
+                    }
+
+                    if (parameters.isAutoExposureLockSupported()) {
+
+                        parameters.setAutoExposureLock(true);
+                    }
+
+                    if (parameters.isAutoWhiteBalanceLockSupported()) {
+
+                        parameters.setAutoWhiteBalanceLock(true);
+                    }
+
+                    _camera.setParameters(parameters);
+
+
+                    _autoFocus = false;
+                } else {
+                    _autoFocus = false;
+                }
```

Comment:
Here you can lift out the `_autoFocus = false;` assignment, because it happens in both blocks, and get rid of the else.